web-registry/src/components/organisms/CreditSendForm/CreditSendForm.schema.tsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { z } from 'zod';
+
+import {
+  invalidRegenAddress,
+  isValidAddress,
+  requiredMessage,
+} from 'web-components/lib/components/inputs/validation';
+
+export const RetireFormSchema = z.object({
+  note: z.string().optional(),
+  country: z.string(),
+  stateProvince: z.string().optional(),
+  postalCode: z.string().optional(),
+  retirementJurisdiction: z.string().optional(),
+});
+
+export type RetireFormSchemaType = z.infer<typeof RetireFormSchema>;
+
+type Params = {
+  addressPrefix?: string;
+};
+
+export const CreditSendFormSchema = ({ addressPrefix }: Params) =>
+  z.object({
+    sender: z.string(),
+    recipient: z
+      .string()
+      .refine(value => !!value, {
+        message: requiredMessage,
+      })
+      .refine(value => isValidAddress(value, addressPrefix), {
+        message: invalidRegenAddress,
+      }),
```

Comment:
We could not put all validation in the zod schema since some rely on several fields at once. We still try to put as much as possible in the schema to give feedback earlier to the user (on onBlur event).